.github/scripts/filter_test_configs.py

```diff
@@ -162,9 +187,10 @@ def main() -> None:
         # No PR number, no tag, we can just return the test matrix as it is
         filtered_test_matrix = test_matrix
 
-    if args.event_name == "schedule":
-        for config in filtered_test_matrix.get("include", []):
-            config["mem_leak_check"] = "mem_leak_check"
+
+    # TODO: DEBUG, to be removed back to == "schedule"
+    if args.event_name != "schedule":
```

Comment:
Once it looks good, I will revert this back to `args.event_name == "schedule"` before merging.